helix-core/src/main/java/org/apache/helix/cloud/AzureInformationFetcher.java

```diff
@@ -0,0 +1,36 @@
+package org.apache.helix.cloud;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import org.apache.helix.common.cloud.CloudInformationFetcher;
+
+public class AzureInformationFetcher implements CloudInformationFetcher {
+  private final String _azureUrl;
+
+  public AzureInformationFetcher(String azureUrl) {
```

Comment:
be more specific? azureUrl is confusing? what does that mean?

Reply:
It's a static url for requesting instance information.
http://169.254.169.254/metadata/instance
I'll think about a better naming.